src/test/regress/expected/multi_cluster_management.out

```diff
@@ -608,7 +608,7 @@ CONTEXT:  PL/pgSQL function citus_internal.pg_dist_node_trigger_func() line 18 a
 INSERT INTO pg_dist_node (nodename, nodeport, groupid, noderole, nodecluster)
   VALUES ('localhost', 5000, 1000, 'primary', 'olap');
 ERROR:  new row for relation "pg_dist_node" violates check constraint "primaries_are_only_allowed_in_the_default_cluster"
-DETAIL:  Failing row contains (19, 1000, localhost, 5000, default, f, t, primary, olap, f).
+DETAIL:  Failing row contains (24, 1000, localhost, 5000, default, f, t, primary, olap, f).
```

Comment:
Note for future readers: The other outputs that have changed incremented by `2`, and this one by `5`, this is expected. 

The reason is that the other changes  done via `master_add_node ` or other UDFs, whereas this it done via a plain `INSERT` to pg_dist_node. For the difference, read the PR description. 

(Finally, see the `multi_cluster_management.sql` file and see that there are 2 proceeding `INSERT INTO pg_dist_node` calls, which is the source for `+2`)